projects.html

```diff
@@ -0,0 +1,388 @@
+						</svg>
+					</div>
+
+					<!-- Hero content -->
+					<div class="relative pt-32 pb-12 md:pt-40 md:pb-20">
+						<!-- Content -->
+						<div class="max-w-3xl mx-auto text-center pb-12 md:pb-16">
+							<h1 class="h1 mb-4" data-aos="fade-up">Our Greatest Hits</h1>
+							<p class="text-xl text-slate-400" data-aos="fade-up" data-aos-delay="200">Or at least the ones we can talk about. We're okay with NDAs, too.</p>
+						</div>
+						<!-- Image -->
+						<img class="mx-auto" src="./images/flow_on_whiteboard.png" width="768" height="432" alt="Hero" data-aos="fade-up" data-aos-delay="400" />
+					</div>
+
+				</div>
+			</section>
+
+			<!-- Stats -->
+			<section>
+				<div class="max-w-6xl mx-auto px-4 sm:px-6">
+					<div class="pb-12 md:pb-20">
+
+						<!-- Section header -->
+						<div class="max-w-3xl mx-auto text-center pb-12 md:pb-16">
+							<h1 class="h2 mb-4">Some Quick Stats</h1>
+							<p class="text-xl text-slate-400">We bring diverse perspectives and diverse experiences to our diverse clientele.</p>
+						</div>
+
+						<div class="grid md:grid-cols-3 bg-slate-800 divide-y md:divide-y-0 md:divide-x divide-slate-700 px-6 md:px-0 md:py-8 text-center">
+							<!-- 1st item -->
+							<div class="py-6 md:py-0 md:px-8">
+								<div class="text-4xl font-bold leading-tight tracking-tighter text-customOrange mb-2" data-aos="fade-up">15+</div>
+								<div class="text-lg text-slate-400" data-aos="fade-up" data-aos-delay="200">Clients on 5 different continents.</div>
+							</div>
+							<!-- 2nd item -->
+							<div class="py-6 md:py-0 md:px-8">
+								<div class="text-4xl font-bold leading-tight tracking-tighter text-customOrange mb-2" data-aos="fade-up">1.000.000</div>
+								<div class="text-lg text-slate-400" data-aos="fade-up" data-aos-delay="200">Lines of code. Not really, but it feels like it.</div>
+							</div>
+							<div class="py-6 md:py-0 md:px-8">
+								<div class="text-4xl font-bold leading-tight tracking-tighter text-customOrange mb-2" data-aos="fade-up">40+</div>
+								<div class="text-lg text-slate-400" data-aos="fade-up" data-aos-delay="200">Years of experience.</div>
```

Comment:
of collective experience is a maybe more transparent

Reply:
done